space.crankshaft.Crankshaft.yml

```diff
@@ -0,0 +1,69 @@
+app-id: space.crankshaft.Crankshaft
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+  - org.freedesktop.Sdk.Extension.node18
+
+command: crankshaft
+
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --talk-name=org.freedesktop.Flatpak
+  # Make $XDG_STATE_HOME work on Flatpak 1.12 and lower
+  - --persist=.local/state
+  - --unset-env=XDG_STATE_HOME
+  # Non-Flatpak Steam
+  - --filesystem=xdg-data/Steam:rw
```

Comment:
Flatpak version of Steam is not yet supported by upstream, but I can add it here once it is